app/helpers/logger/logger.js

```diff
@@ -18,31 +18,35 @@ const Logger = function (debugging, externalLogger, fileSystem) {
 
   };
 
-  const error = function (detail, errorDetail) {
+  const getMessage = function (type, detail, fileUrl) {
+    return type + ": " + detail + ((fileUrl) ? " - " + fileUrl : "");
```

Comment:
I think we can use template literals here - https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Template_literals